Automattic/wp-calypso
# Unify/titan notice components into one refactored component

#### Changes proposed in this Pull Request

This is the second step to unify the GoogleMailboxPricingNotice and TitanMailboxPricingNotice into one single component called MailBoxPricingNotice.

As a second step it will remove the GoogleMailboxPricingNotice and the TitanMailboxPricingNotice, as they won't be used anymore. Also the component has been expanded to support monthly/annually labels.

#### Testing instructions

This test should be tested for Titan monthly & Titan yearly subscriptions.
As regression test, we should test that the test for this [pull request](https://github.com/Automattic/wp-calypso/pull/61566) still work

1. Apply this patch locally or use the calypso live branch
2. Go to Upgrades -> Email and add a mailbox to a current Titan subscription (if you don't have you should add a subscription to a domain)
3. Check that you are presented with the following screen, with a notice in the top of the form.

You should be presented with this screen:
![image](https://user-images.githubusercontent.com/5689927/156193109-1d541ed7-8a2f-410e-b43c-c7a92f7514ca.png)

Related to {1200182182542585-as-1201900052374135}
